docs/client_reference.rst

```diff
@@ -250,8 +271,7 @@ request coroutine
                        read_until_eof=True, request_class=None,\
                        response_class=None)
 
-   Perform an asynchronous http request. Return a response object
-   (:class:`ClientResponse` or derived from).
+   Performs an asynchronous http request. Returns a response object.
```

Comment:
The below changes in this file are side effects of the merge, pollutes the history a bit. Maybe a git rebase will solve this next time. Sorry a bit new the pull ecosystem.